engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/history/HistoricJobLogRestServiceQueryTest.java

```diff
@@ -312,6 +312,16 @@ public void testSortingParameters() {
     executeAndVerifySorting("tenantId", "desc", Status.OK);
     inOrder.verify(mockedQuery).orderByTenantId();
     inOrder.verify(mockedQuery).desc();
+
+    inOrder = Mockito.inOrder(mockedQuery);
+    executeAndVerifySorting("hostname", "asc", Status.OK);
+    inOrder.verify(mockedQuery).orderByTenantId();
+    inOrder.verify(mockedQuery).asc();
+
+    inOrder = Mockito.inOrder(mockedQuery);
+    executeAndVerifySorting("hostname", "desc", Status.OK);
+    inOrder.verify(mockedQuery).orderByTenantId();
```

Comment:
This has to be changed, right?
```suggestion
    inOrder.verify(mockedQuery).orderByHostname();
```